src/elements/time-input/time-input.visual.spec.ts

```diff
@@ -21,11 +21,17 @@ describe(`sbb-time-input`, () => {
     withError: [false, true],
   };
 
+  const sizeCases = {
+    size: ['s', 'm', 'l'],
+    noIcons: [false, true],
+  };
+
   type ParamsType = { [K in keyof typeof cases]: (typeof cases)[K][number] } & {
+    [K in keyof typeof sizeCases]: (typeof sizeCases)[K][number];
+  } & {
     readonly?: boolean;
     borderless?: boolean;
     disabled?: boolean;
-    noIcons?: boolean;
   };
   const template = (args: Partial<ParamsType>): TemplateResult => html`
```

Comment:
size needs to be respected. right now, all visual regression tests have the same size